.github/ISSUE_TEMPLATE/00-project-onboarding-checklist-template.md

```diff
@@ -9,22 +9,22 @@ assignees: ''
 
 ## Project Onboarding Checklist - ${Project Name}
 
-<!--Copied from ../../PROJECT_PROGRESSION.md on 2019-08-23 -->
+<!--Copied from ../../PROJECT_PROGRESSION.md on 2019-10-31 -->
 
 **Stage: ${Project Stage}
 
 - [ ] Adopt the OpenJS Foundation Code of Conduct
 - [ ] Update project CoC reporting methods to include OpenJS Foundation escalation path
-- [ ] Transfer official domains to OpenJS Foundation
+- [ ] List official domains that the project commits to transfer to the OpenJS Foundation following graduation
 - [ ] Identify and document other core project infrastructure
-- [ ] If choosing to use a Contributor License Agreement (CLA) or Developer Certificate of Origin (DCO), make selection and implement appropriate tool
+- [ ] If choosing to use a Contributor License Agreement (CLA) or Developer Certificate of Origin (DCO), make selection
```

Comment:
```suggestion
- [ ] If choosing to use a Contributor License Agreement (CLA) or Developer Certificate of Origin (DCO), choose one, or "None"
```

Reply:
Per the IP policy, projects should implement the DCO.  The CLA is optional on a per-project basis, though.

Reply:
Why is this the IP policy? https://ben.balter.com/2018/01/02/why-you-probably-shouldnt-add-a-cla-to-your-open-source-project/ seems relevant.

Reply:
My understanding is that the JSF required all projects to use a CLA (@kborchers may be able to shed some light on the roots of that policy).  By contrast, the Node.js Foundation did not.

The IP Policy strikes a balance between the two by making the CLA optional going forward.  Projects who have a reason to use it may continue to do so, whereas those who either no longer wish to use it or who never had it in the first place can just use DCO.

Reply:
From what https://developercertificate.org implies, this is already how github works implicitly. What does the DCO requirement impose on contributors, maintainers, and projects?

Reply:
Suggested a session on these IP-related issues at the collab summit next week [here](https://docs.google.com/document/d/1hdyWxXjCXUe1iEUqaQE8jvt38N6ckQomxTCtgkdisnU/edit). Additional subtopics welcomed.

Reply:
So the current IP policy won't let us choose "none" as an option. I won't be merging that suggestion, @ljharb, and suggest we continue this discussion in https://github.com/openjs-foundation/cross-project-council/issues/387.